### PR TITLE
Creation of cat.md for pytorch

### LINKED ISSUE #5350: [Term Entry] PyTorch Tensors .cat()
### Reason/inspiration (optional)

We would like a new entry on the `.cat()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensors/terms/cat/cat.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is an attempt to create the .cat() entry for PyTorch. This aims to provide additional information to the body of knowledge on the topic.

### Issue Solved

Closes #5350 

### Type of Change

- Adding a new entry

### Checklist

- [/] All writings are my own.
- [/] My entry follows the Codecademy Docs style guide.
- [/] My changes generate no new warnings.
- [/] I have performed a self-review of my own writing and code.
- [/] I have checked my entry and corrected any misspellings.
- [/] I have made corresponding changes to the documentation if needed.
- [/] I have confirmed my changes are not being pushed from my forked `main` branch.
- [/] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [/] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by arisdelacruz** [comment]
@mamtawardhani 

I think I got it right. Thanks

File: content/pytorch/concepts/tensors/terms/cat/cat.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+---
+
+The **`.cat()`** function in PyTorch is used for concatenating two or more tensors in the same dimension. The tensors must have the same shape in all dimensions except the one in which they are concatenated.
+
+## Syntax
+
+```pseudo
+torch.cat(tensors, dim=0)
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/cat/cat.md:22]: Can we also mention the out parameter here? I think it would be helpful. Example:

torch.cat(tensors, dim=0, out=None)

File: content/pytorch/concepts/tensors/terms/cat/cat.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+```
+
+## Parameters
+
+- **tensors** (sequence of Tensors) – The tensors to concatenate.
+- **dim** (int) – The dimension along which the tensors are concatenated.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/cat/cat.md:28]:
```suggestion
- `tensors`: A sequence of tensors to be concatenated. All tensors must have the same shape in all dimensions except for the specified dimension.
- `dim`: An integer specifying the dimension along which the tensors will be concatenated. The default value is 0, which means concatenation will occur along the first dimension.
```

Comment (mamtawardhani) [reply content/pytorch/concepts/tensors/terms/cat/cat.md:28]: if you add the `out` in syntax, please do add it here as well :)

File: content/pytorch/concepts/tensors/terms/cat/cat.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+## Example
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/cat/cat.md:30]:
```suggestion
## Example 1
```

File: content/pytorch/concepts/tensors/terms/cat/cat.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+## 1. Concatenating tensors along the first dimension
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/cat/cat.md:32]:
```suggestion
The example below showcases concatenating tensors along the first dimension using the `.cat()` function:
```

File: content/pytorch/concepts/tensors/terms/cat/cat.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+        [10, 11, 12]])
+
+result:
+tensor([[ 1,  2,  3],
+        [ 4,  5,  6],
+        [ 7,  8,  9],
+        [10, 11, 12]])
+```
+
+## 2. Concatenating tensors along the second dimension
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/cat/cat.md:72]:
```suggestion
## Example 2

The example below showcases concatenating tensors along the second dimension using the `.cat()` function:
```

File: content/pytorch/concepts/tensors/terms/cat/cat.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+tensor2:
+tensor([[ 7,  8,  9],
+        [10, 11, 12]])
+
+result:
+tensor([[ 1,  2,  3,  7,  8,  9],
+        [ 4,  5,  6, 10, 11, 12]])
+```
+
+## 3. Concatenating tensors along the third dimension
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/cat/cat.md:110]:
```suggestion
## Example 3

The example below showcases concatenating tensors along the third dimension using the `.cat()` function:
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @arisdelacruz thank you for contributing to Codecademy Docs! The entry is very well written! 😄 

I've suggested some minor changes, could you please review and modify those at your earliest convenience? Thank you! 😃

**Comment by arisdelacruz** [comment]
done, @mamtawardhani 

thanks

**Review by mamtawardhani — APPROVED** [review]
Thank you for the modifications @arisdelacruz 😄 

The entry looks good for a second round of review 🚀

**Review by Radhika-okhade — APPROVED** [review]
Thank you for contributing to Codecademy docs. The entry looks good to be merged.
